<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1116</width>
    <height>659</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(248, 250, 212);</string>
  </property>
  <property name="iconSize">
   <size>
    <width>32</width>
    <height>32</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>541</width>
      <height>591</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Генератор текста</string>
    </property>
    <widget class="QTextEdit" name="textEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>271</width>
       <height>481</height>
      </rect>
     </property>
     <property name="cursor" stdset="0">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Введите свой текст, нажмите кнопку&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="whatsThis">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Введите свое предложение, нажмите на кнопку, оно будет дополнено&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 9pt &quot;Verdana&quot;;
background-color: rgb(255, 255, 255);
text-align: justify;
</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>30</y>
       <width>241</width>
       <height>221</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Verdana</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Параметры дополнения</string>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
        <pointsize>10</pointsize>
        <weight>9</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 10pt;</string>
      </property>
      <property name="text">
       <string>Кол-во слов:</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="minimum">
       <number>10</number>
      </property>
      <property name="maximum">
       <number>60</number>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="editable">
       <bool>true</bool>
      </property>
      <item>
       <property name="text">
        <string>Русский</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>English(beta)</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 10pt;</string>
      </property>
      <property name="text">
       <string>Язык</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>131</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 10pt;</string>
      </property>
      <property name="text">
       <string>Тематика текста</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_2">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>180</y>
        <width>211</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="editable">
       <bool>true</bool>
      </property>
      <item>
       <property name="text">
        <string>Авто(остальные в доработке)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Русский язык</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Математика</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Физика</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Правоведение</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Экономика</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>История</string>
       </property>
      </item>
     </widget>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>520</y>
       <width>191</width>
       <height>61</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#pushButton {
	background-color: rgb(255, 148, 150);
	border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    font: bold 14px;
    min-width: 10em;
    padding: 6px;
}

QPushButton#pushButton:pressed{
	background-color: rgb(212, 155, 139);
}
</string>
     </property>
     <property name="text">
      <string>Сгенерировать
текст</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>310</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 10pt;</string>
     </property>
     <property name="text">
      <string>Кол-во символов:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>270</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 12pt &quot;Verdana&quot;;
text-decoration: underline bold;
border-color: rgb(191, 28, 255);
background-color: rgb(255, 223, 60);</string>
     </property>
     <property name="text">
      <string>Состояние текста</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>340</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 10pt;</string>
     </property>
     <property name="text">
      <string>Кол-во слов:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>370</y>
       <width>171</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 10pt;</string>
     </property>
     <property name="text">
      <string>Кол-во предложений:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_Symb">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>310</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 10pt;</string>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_Word">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>340</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 10pt;</string>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_Sent">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>370</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 10pt;</string>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>430</y>
       <width>190</width>
       <height>31</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#pushButton_3 {
	background-color: rgb(255, 148, 150);
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    font: bold 14px;
    min-width: 10em;
    padding: 6px;
}

QPushButton#pushButton_3:pressed{
	background-color: rgb(212, 155, 139);
}
</string>
     </property>
     <property name="text">
      <string>Варианты дополнения</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>480</y>
       <width>182</width>
       <height>28</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>182</width>
       <height>0</height>
      </size>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#pushButton_4 {
	background-color: rgb(255, 148, 150);
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    font: bold 14px;
    min-width: 10em;
    padding: 6px;
}

QPushButton#pushButton_4:pressed{
	background-color: rgb(212, 155, 139);
}
</string>
     </property>
     <property name="text">
      <string>Очистить</string>
     </property>
    </widget>
    <zorder>groupBox_2</zorder>
    <zorder>textEdit</zorder>
    <zorder>pushButton</zorder>
    <zorder>label_5</zorder>
    <zorder>label_6</zorder>
    <zorder>label_7</zorder>
    <zorder>label_8</zorder>
    <zorder>label_Symb</zorder>
    <zorder>label_Word</zorder>
    <zorder>label_Sent</zorder>
    <zorder>pushButton_3</zorder>
    <zorder>pushButton_4</zorder>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>10</y>
      <width>541</width>
      <height>591</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Перепись печатного в рукописный</string>
    </property>
    <widget class="QTextEdit" name="textEdit_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>271</width>
       <height>481</height>
      </rect>
     </property>
     <property name="cursor" stdset="0">
      <cursorShape>IBeamCursor</cursorShape>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Введите текст, нажмите кнопку, сохраните изображение&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="whatsThis">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Введите текст, нажмите кнопку, сохраните изображение&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 9pt &quot;Verdana&quot;;
background-color: rgb(255, 255, 255);
text-align: justify;

</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>520</y>
       <width>191</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Verdana</family>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#pushButton_2 {
	background-color: rgb(255, 148, 150);
	border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    font: bold 14px;
    min-width: 10em;
    padding: 6px;
}

QPushButton#pushButton_2:pressed{
	background-color: rgb(212, 155, 139);
}
</string>
     </property>
     <property name="text">
      <string>Перевести 
в рукописный</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_image">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>130</y>
       <width>221</width>
       <height>341</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;предпросмотр страницы с&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;рукописным текстом будет&lt;/p&gt;&lt;p align=&quot;center&quot;&gt; здесь&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>40</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 10pt;</string>
     </property>
     <property name="text">
      <string>Размер шрифта</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>100</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 10pt;</string>
     </property>
     <property name="text">
      <string>Шрифт</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>70</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 10pt;</string>
     </property>
     <property name="text">
      <string>Цвет шрифта</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_2">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>40</y>
       <width>50</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="minimum">
      <number>8</number>
     </property>
     <property name="maximum">
      <number>20</number>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_5">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>520</y>
       <width>191</width>
       <height>60</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#pushButton_5 {
	background-color: rgb(255, 148, 150);
	border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    font: bold 14px;
    min-width: 10em;
    padding: 6px;
}

QPushButton#pushButton_5:pressed{
	background-color: rgb(212, 155, 139);
}
</string>
     </property>
     <property name="text">
      <string>Сохранить файл</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_3">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>70</y>
       <width>73</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <item>
      <property name="text">
       <string>Черный</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Синий</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Голубой</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="comboBox_4">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>100</y>
       <width>73</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Gogol</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="pushButton_6">
     <property name="geometry">
      <rect>
       <x>343</x>
       <y>480</y>
       <width>186</width>
       <height>28</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#pushButton_6 {
	background-color: rgb(255, 148, 150);
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    font: bold 14px;
    min-width: 10em;
    padding: 6px;
}

QPushButton#pushButton_6:pressed{
	background-color: rgb(212, 155, 139);
}
</string>
     </property>
     <property name="text">
      <string>Очистить</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>610</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 italic 18pt &quot;Verdana&quot;;
text-decoration: underline;</string>
    </property>
    <property name="text">
     <string>Михаил К. 2021</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>610</y>
      <width>531</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 18pt &quot;Verdana&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <action name="action">
   <property name="text">
    <string>Использование</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>О разработчике</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
